wwwroot/buy-anything.html

```diff
@@ -0,0 +1,414 @@
+                                        <p>Enjoy free shipping within the EU, access millions of products, and uncover incredible travel deals while Buying Anything with Bitcoin on Wasabi Wallet.</p>
+                                    </div>
+                                </div>
+                                <div class="col-12 mb-4">
+                                    <div class="feature-box">
+                                        <div class="title my-3 green-text">3. Privacy-First Shopping</div>
+                                        <p>Wasabi Wallet and ShopinBit prioritize your privacy, sharing only the necessary information for order fulfillment.</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+
+                </div>
+            </section>
+            <section class="home p-4 green-bg">
+                <div class="container mb-5">
+                    <div class="row align-items-center">
+                        <div class="col-12 col-md-6">
+                            <div class="my-5">
+                                <h2 class="pt-header title-lined left">Exclusive Rewards</h2>
+                                <p class="big-text">Don't miss out on our exclusive offer! Random buyers using the new feature stand a chance to win <strong>$100 vouchers</strong>. Act fast, offer ends December 14th XX:XX UTC.</p>
+                            </div>
+                        </div>
+                        <div class="col-md-6">
+                            <img src="img/christmastree_2.png" class="w-100">
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="home p-4">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-12">
+                            <div class="my-5">
+                                <h2 class="pt-header title-lined left">Frequently Asked Questions</h2>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row mb-5">
+                        <div class="col-12">
+                            <div class="card">
+                                <input type="checkbox" id="faq1">
+                                <label for="faq1">
+                                    1. Can I really buy anything?
+                                </label>
+                                <div class="card-content">
+                                    <p>Shopinbit's Concierge Service does not provide any kind of drugs, prescription medications, weapons, precious metals, cryptocurrencies, financial instruments, or any type of product that is illegal in Poland. You can buy anything else.</p>
```

Comment:
It's at some places "ShopinBit", at others "Shopinbit", should be consistent everywhere.

Reply:
*should be _Shopinbit_ everywhere, that's what they use themselves